nus-faster-pay/src/Screens/QRCode/index.tsx

```diff
@@ -1,13 +1,38 @@
+import React, { useEffect } from 'react';
 import payLahImage from "./images/pay_lah.jpg"; // Import the image correctly
```

Comment:
button needed here to go back to StartScreen!

Reply:
created navigation